<launch>
    <node pkg="vb_greedy_planner" type="vb_greedy_planner_node" name="vb_greedy_planner_node" output="screen">
        <rosparam command="load" file="$(find vb_greedy_planner)/config/default.yaml" />
    </node>
</launch>